doc/stdlib/io-network.rst

```diff
@@ -168,11 +168,11 @@ General I/O
 
    See ``read`` for a description of the ``all`` option.
 
-.. function:: read(stream::IO, nb=typemax(Int); all=true)
+.. function:: read(s::IOStream, nb::Integer; all=true)
```

Comment:
you can't, so maybe the signature with `nb=typemax(Int)` default value but no kwarg should be a separate docstring?